packages/components/psammead-consent-banner/src/index.jsx

```diff
@@ -33,108 +51,95 @@ const Wrapper = styled.div`
   @media (min-width: ${GEL_GROUP_2_SCREEN_WIDTH_MIN}) {
     padding: ${GEL_MARGIN_ABOVE_400PX};
   }
+
+  @media (min-width: ${GEL_GROUP_3_SCREEN_WIDTH_MIN}) {
+    padding-top: ${GEL_SPACING_QUAD};
+    padding-bottom: ${GEL_SPACING_QUAD};
+  }
 `;
 
-/*
- * The '&::after' below is to ensure that the background colour covers the
- * banner as the inner elements are float. The alernative is to have
- * another div inside. This implementation mirrors the current orbit banner.
- */
 const CenterWrapper = styled.div`
-  max-width: ${GEL_GROUP_5_SCREEN_WIDTH_MIN};
+  max-width: 36.25rem;
```

Comment:
Echoing what Andrew has said – let's try and reuse a constant closest to the Zeplin width rather than introducing an arbitrary one. 👍🏼